### PR TITLE
[Comparison] Update comparison table and UI

### PR DESCRIPTION
**Pull request by ibelem**
The comparison table got some positive feedback from Google and China community, fix some text issues and update the page UI.

Please refer to https://ibelem.github.io/mini-app-white-paper/comparison.html 

@Honry @xueyuanjia @xfq PTAL

### CODE REVIEW

**Comment by Honry** [comment]
@ibelem, could you provide a preview? Just push this commit to `ibelem:gh-pages` branch, thus we will see the new UI at https://ibelem.github.io/mini-app-white-paper/comparison.html

**Comment by ibelem** [comment]
@Honry done

File: comparison.html [hunk 1]
```diff
@@ -1,2249 +1,1507 @@
...
-
-
-
-
-
-
-
-          </tr>
-          <tr class="row90">
-            <td class="column0 style6 s">Security and<br
```
Comment (Honry) [inline comparison.html]: Missing a `)`.

Comment (Honry) [inline comparison.html]: Missing a `)`.

Comment (Honry) [inline comparison.html]: Missing a `)`.

**Comment by xfq** [comment]
The color contrast on hyperlinks in the "W3C Spec" column when hovering other columns isn't quite good. Do you mind making the color lighter, @ibelem?

**Comment by xfq** [comment]
It would be good to add `min-width` and/or `padding-{left,right}` on `th` elements too, because otherwise the table header might become too narrow sometimes (search "NFC" and see the "W3C Spec" header as an example).

**Comment by xfq** [comment]
The `cellpadding="0" cellspacing="0" border="0"` attributes on `table` can be removed?

**Comment by ibelem** [comment]
Thanks @Honry and @xfq , all issues above have been fixed. https://ibelem.github.io/mini-app-white-paper/comparison.html

**Pull request merged**